terraform/addons/byo-firehose-logging-destination/target-account/s3.tf

```diff
@@ -1,7 +1,7 @@
 data "aws_region" "current" {}
 data "aws_caller_identity" "current" {}
 data "aws_kms_alias" "s3" {
-  name = "aws/s3"
```

Comment:
interesting because some folks have already deployed this..

Reply:
![image](https://github.com/fleetdm/fleet/assets/1945008/88c7142d-b641-47b2-8453-1a5fd988e79b)